src/mkdocstrings/handlers/__init__.py

```diff
@@ -110,20 +111,24 @@ def __init__(self, directory: str, theme: str) -> None:
         Arguments:
             directory: The name of the directory containing the themes for this renderer.
             theme: The name of theme to use.
+            custom_templates: Directory containing custom templates.
         """
+
+        self.directory = directory
+
+        paths = []
+
+        if custom_templates is not None:
+            paths.append(Path(custom_templates) / directory / theme)
+
         themes_dir = Path(__file__).parent.parent / "templates" / directory
-        theme_dir = themes_dir / theme
-        if not theme_dir.exists():
-            if self.FALLBACK_THEME != "":
-                log.warning(
-                    f"mkdocstrings.handlers: No '{theme}' theme in '{directory}', "
-                    f"falling back to theme '{self.FALLBACK_THEME}'"
-                )
-                theme_dir = themes_dir / self.FALLBACK_THEME
-            else:
-                raise ThemeNotSupported(theme)
-
-        self.env = Environment(autoescape=True, loader=FileSystemLoader(str(theme_dir)))
+
+        paths.append(themes_dir / theme)
+
+        if self.FALLBACK_THEME != "" and False:
```

Comment:
Why the `and False`? This will never be true :confused:

Reply:
Because I was testing something and I forgot to remove it...